src/ZendDiagnostics/Check/CouchDBCheck.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendDiagnostics\Check;
+
+use ZendDiagnostics\Result;
+
+/**
+ * Ensures a connection to CouchDB is possible.
+ */
+class CouchDBCheck extends GuzzleHttpService
+{
+    /**
+     * @param array $couchDbSettings
+     *
+     * @return self
+     */
+    public function __construct(array $couchDbSettings)
+    {
+        $couchDbUrl = '';
+        if ($couchDbSettings['port'] === '5984' || $couchDbSettings['port'] === '80') {
```

Comment:
I guess these are the defaults but I assume this stuff is configurable, so for some users this might not work .. right?